src/client.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * @title Assembly API
+ * @version 2.0
+ * @baseUrl https://virtserver.swaggerhub.com/AssemblyPlatforms/assembly-api/2.0
```

Comment:
The link isn't working.
Should be `https://api.swaggerhub.com/apis/AssemblyPlatforms/assembly-api/2.0/swagger.yaml`?